Core/MIPS/ARM64/Arm64CompVFPU.cpp

```diff
@@ -654,12 +654,58 @@ namespace MIPSComp {
 				switch ((op >> 23) & 7) {
 				case 2:  // vmin
 				{
+					fp.FCMP(fpr.V(sregs[i]), fpr.V(tregs[i]));
+					FixupBranch unordered = B(CC_VS);
 					fp.FMIN(fpr.V(tempregs[i]), fpr.V(sregs[i]), fpr.V(tregs[i]));
+					FixupBranch skip = B();
+
+					SetJumpTarget(unordered);
+					// Move to integer registers, it'll be easier.  Or maybe there's a simd way?
```

Comment:
For rarely used chunks of code like this, with mostly fixed registers, we might benefit from pre-generating it in JitAsm and calling it? Or, put it in Dolphin-style "far code" to reduce instruction cache pollution. Just a thought for the future.